src/mg/PAMI/Client/Impl/ClientImpl.php

```diff
@@ -405,20 +405,21 @@ public function send(OutgoingMessage $message)
 	    if (@fwrite($this->_socket, $messageToSend) < $length) {
     	    throw new ClientException('Could not send message');
 	    }
-	    $read = 0;
-	    while($read <= $this->_rTimeout) {
-	        $this->process();
```

Comment:
If you remove this line, how are events handled? The ones not related to a response